helm-system-packages-portage.el

```diff
@@ -75,33 +65,29 @@
     (helm-update)))
 (put 'helm-system-packages-portage-toggle-dependencies 'helm-only t)
 
+(defvar helm-system-packages-portage-map
+  ;; M-U is reserved for `helm-unmark-all'.
+  (let ((map (make-sparse-keymap)))
+    (set-keymap-parent map helm-map)
+    (define-key map (kbd "M-I")   'helm-system-packages-portage-toggle-explicit)
+    (define-key map (kbd "M-N")   'helm-system-packages-portage-toggle-uninstalled)
+    (define-key map (kbd "M-D")   'helm-system-packages-portage-toggle-dependencies)
```

Comment:
Those 3 -toggles can be replaced with the package-level -toggles, e.g. `helm-system-packages-toggle-explicit`.